tools/embedded_data_analyzer.py

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/python3
```

Comment:
Unfortunately, this won't work on every distribution. It's best to use the `python3` interpreter from the PATH environment:

```suggestion
#! /usr/bin/env python3
```

Reply:
Thanks for the note, this changed has been added.